src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,275 @@
+
+MindShields are metagameable and make it extremely unfun to play as a Head Rev. They have no interesting interactions; they’re just a hand-wave for why some people are completely immune to a game mechanic, and whilst that’s utterly required for round flow (allowing Sec and the Heads to trust each other) it is a magic solve-everything button which leads to a total wall in design.
+
+At best, a small number of MindShields could be kept on-station to allow staff promotions, and if these run out then more can be begged for via asking CentComm nicely.
+
+### Removal of Flashes being able to convert people
+
+Flashes are metagameable, confusing for newbies, hard to balance, and enforce too much reliance on manufacture by Science by the Revs for effective round flow. They also have no narrative justification beyond it being an Ur-mode from the dawn of SS13. There are people playing SS14 that are younger than the only good _Men In Black_ movie.
+
+### Addition of Influence as a tracked player stat, and associated gameplay features
+
+Influence is a value that each player carries. MindShielded players are always value 0. Influence is raised by a few routes (exposure to Head Revs, propaganda and one or two restricted drugs like Mindbreaker Toxin).
+
+### Influence ticks up when exposed to sources and slowly ticks down when not. There is a max Influence level.
+
+Addition of NT and Revolutionary posters and propaganda, that both sides, and traitors, can put up and take down across the station.
+
+These posters can be printed across the station by both sides and gives Security Cadets, Lawyers and bored Passengers something to do, even on non-Rev rounds, and a reason for Sec to patrol (imagine!). They’re an DOT effect with a max cap per item, and add or subtract Influence from those who see them.
+
+### Addition of Loyalty Loudspeakers in Sec
+
+These Loyalty Loudspeakers are audio emitters that reduce a player’s Influence level, even if they’re converted. If a player who is converted de-converts, their Influence immediately drops to zero. This is a new machine that requires quite a bit of power to run. A button is added to the Warden’s office to disable and enable all loudspeakers. The loudspeakers play calming, smooth jazz.
+
+### Head Revs can use an implant ability to silently convert characters with high enough Influence to being a Rev.
+
+This replaces flashing outright, and makes conversion a stealthy game. Revs do not know who the Head Revs are unless the Head Rev outs themselves.
+
+### Being a Rev has a tell, similar to some old SS13 conversion modes.
+
+Being a Rev means you look slightly dazed and confused, which can be hidden by hiding your eyes. This effect will also be applied to some drugs in game like Mindbreaker Toxin.
+
+### Killing the Heads is now not required for the Revs to win, instead requiring taking the Bridge and using all Head ID cards on the Command Console and defending the Bridge for a period of time.
```

Comment:
Good comments. One alternative for the cards I was considering was a retinal scan of each of the recognized Heads (meaning you can gib, murder or capture a Head and drag 'em to the Bridge) but I went for IDs as it's the universal security/ID chit.

For the capture-the-point problem: I'm sort of fumbling around with what exact terminal to use. One "uber terminal" that reflects the core Centcomm uplink is probably what would have to be needed, which would probably be made of completely indestructible materials. Like a couple of other points so far it makes me :/ as it is a bit video game-y

Reply:
What if for the round to end every "Head" (or someone with that head's access) must activate some thing on their office's computer terminal. Once a majority of the buttons are on, the bridge comms console then has a final button to press to have the station succeed from NT. This would mean the revs need to hold territory in the form of each head office and the bridge and if they allow defending loyalists to take over a majority again and disable them, they must take it back. 

Revs Start > Take control of over 50% of Head Offices > Activate Computer Button > Take control of bridge > Activate comms console button > Win